### PR TITLE
Add Visualping

### PR DESCRIPTION
**Pull request by SteffenGivard**
Add [Visualping](https://visualping.io/) as having "limited" account deletion.

I wasn't asked to show any proof of living in a location with privacy rights.

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
> I wasn't asked to show any proof of living in a location with privacy rights.

But did they ask where you were from or something? Or you just requested for deletion via chat/email and they followed through?

**Comment by SteffenGivard** [comment]
After deactivating my account, I went and filled out [their contact form](https://visualping.io/contact/). I choose to contact support and wrote "I'd like to have my account deleted in accordance to the GDPR.". Some time later I got a confirmation mail about my account being deleted.

File: _data/sites.json [hunk 1]
```diff
@@ -19442,6 +19442,16 @@
         ]
     },
 
+    {
+        "name": "Visualping",
+        "url": "https://visualping.io/account/profile/",
+        "difficulty": "limited",
```
Comment (tupaschoal) [inline _data/sites.json:19448]:
```suggestion
        "difficulty": "hard",
```

**Review by tupaschoal — COMMENTED** [review]
I'll change it to hard then, since they didn't ask for proof

**Pull request merged**